ReactAndroid/src/main/java/com/facebook/react/views/androidtv/ReactAndroidTVRootViewHelper.java

```diff
@@ -0,0 +1,112 @@
+package com.facebook.react.views.androidtv;
+
+import android.view.KeyEvent;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.facebook.react.ReactRootView;
+import com.facebook.react.bridge.WritableMap;
+import com.facebook.react.bridge.WritableNativeMap;
+import com.facebook.react.modules.core.DeviceEventManagerModule.RCTDeviceEventEmitter;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Responsible for dispatching events specific for Android TV to support D-PAD navigation/selection.
+ * This is similar to AppleTV implementation and uses the same emitter events.
+ */
+public class ReactAndroidTVRootViewHelper {
+
+  /**
+   * Android TV remote control sends a DPAD_CENTER event when clicking on a focused item.
+   * We add ENTER and SPACE to facilitate navigating on Android TV emulator.
+   */
+  private static final List<Integer> PRESS_KEY_EVENTS = Arrays.asList(
+    KeyEvent.KEYCODE_DPAD_CENTER,
+    KeyEvent.KEYCODE_ENTER,
+    KeyEvent.KEYCODE_SPACE
+  );
+
+  /**
+   * We keep reference to the last focused view id so that we can send a blur event when focus changes.
+   */
+  private int mLastFocusedViewId = View.NO_ID;
+
+  private ReactRootView mReactRootView;
+
+  public ReactAndroidTVRootViewHelper(ReactRootView reactRootView) {
+    mReactRootView = reactRootView;
+  }
+
+  /**
+   * Called from {@link ReactRootView}.
+   * This is the main place the Android TV remote key events are handled.
+   */
+  public void handleKeyEvent(KeyEvent ev, RCTDeviceEventEmitter emitter) {
+    int eventKeyCode = ev.getKeyCode();
+    int eventKeyAction = ev.getAction();
+    View targetView = findFocusedView(mReactRootView);
+    if (targetView != null) {
+      if (KeyEvent.KEYCODE_MEDIA_PLAY_PAUSE == eventKeyCode && eventKeyAction == KeyEvent.ACTION_UP) {
```

Comment:
@kmagiera Now that I think about it, maybe we should dispatch the "playPause" event event without having a focused view? It does not include the view id after all.
At first I thought that it would make no sense to send playPause events without having any view focused, but maybe I'm wrong.